desktop/apps/auction/components/layout/auction_info/Registration.js

```diff
@@ -16,10 +18,14 @@ function Registration (props) {
 
   const b = block('auction-Registration')
 
+  if (isEcommerceSale) {
+    return null
+  }
+
   return (
     <div className={b()}>
       {(() => {
-        if (isClosed) {
+        if (isClosed || isLiveOpen) {
```

Comment:
I think we still want to show the registration button if the sale is live (there is the possibility that registration doesn't end, although it is uncommon).